control-plane/csi-driver/src/bin/controller/controller.rs

```diff
@@ -799,7 +828,8 @@ impl rpc::csi::controller_server::Controller for CsiControllerSvc {
         }))
     }
 
-    #[instrument(err, fields(volume.uuid = request.get_ref().source_volume_id, snapshot.source_uuid = request.get_ref().source_volume_id, snapshot.uuid), skip(self))]
+    #[instrument(err, fields(volume.uuid = request.get_ref().source_volume_id, snapshot.source_uuid = request.get_ref().source_volume_id, snapshot.uuid
```

Comment:
what's modified here?

Reply:
Ohh nothing, the formatter did this, it seems.